website/siteConfig.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+// List of projects/orgs using your project for the users page.
+const users = [
+  {
+    caption: 'User1',
+    // You will need to prepend the image path with your baseUrl
+    // if it is not '/', like: '/test-site/img/image.jpg'.
+    image: '/img/undraw_open_source.svg',
+    infoLink: 'https://www.facebook.com',
+    pinned: true
+  }
+];
+
+const siteConfig = {
+  title: 'OFFIX', // Title for your website.
+  tagline:
+    'Offix extends the capabilities of Apollo GraphQL providing a fully featured Offline Workflow and Conflict Resolution.',
+  url: 'https://offix.dev', // Your website URL
+  baseUrl: '/', // Base URL for your project */
+  // For github.io type URLs, you would set the url and baseUrl like:
+  //   url: 'https://facebook.github.io',
+  //   baseUrl: '/test-site/',
+
+  // Used for publishing and more
+  projectName: 'OFFIX',
+  organizationName: 'Aerogear',
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: 'gettingStarted', label: 'Documentation' }
+    // { blog: true, label: 'Blog' }
+  ],
+
+  // If you have users set above, you add it here:
+  users,
+
+  /* path to images for header/footer */
+  headerIcon: 'img/offix-logo.png',
+  footerIcon: 'img/offix-logo.png',
+  favicon: 'img/offix-logo.png',
```

Comment:
oh.. this will be overkill. We can create separate pr will actual favicon

Reply:
I didn't get what you want in here you want me to change file format? I have snipped from offix.dev and removed the background 😀😀

Reply:
Favicons are usually 32px etc. smal factor. Ignore me as this is something we can do after